filebeat/registrar/migrate.go

```diff
@@ -119,31 +171,110 @@ func initRegistry(regHome string, perm os.FileMode) error {
 	return nil
 }
 
-func readVersion(regHome, migrateFile string) (string, error) {
+// migrateVersion1 migrates the filebeat registry from version 0 to version 1
+// only. Version 1 is based on the implementation of version 1 in
+// libbeat/registry/backend/memlog.
+func migrateVersion1(home, regHome string, perm os.FileMode) error {
+	logp.Info("Migrate registry version 0 to version 1")
+
+	origDataFile := filepath.Join(regHome, "data.json")
```

Comment:
Nit: Could you please rename `origDataFile` to `originalDataFile`. It is a bit confusing to me. I was not sure whether it is supposed to be `original` or `origin`.